teamwork/templates/projects/edit_project.html

```diff
@@ -305,6 +305,15 @@ <h3 class="box-title">Project Members</h3>
 <div class="col-md-6">
   <div class="box box-primary">
     <div class="box-header with-border">
+        <div class="form-group">
```

Comment:
Lock request to join shouldn't be in the same div as the box-title.

![image](https://user-images.githubusercontent.com/6921374/52010194-c07c4580-2489-11e9-96e7-9f531f5b7893.png)